sunspec.go

```diff
@@ -28,10 +28,125 @@ type ScaleFactor int16
 
 type Count uint16
 
+// An array is a collection of devices.
+//
+// This type is used when the underlying representation
+// is an XML document.
+type Array interface {
+	Device(id string) Device // Answer the specified device instance or nil
+	Do(func(d Device))       // Iterate over all the devices.
+}
```

Comment:
Is there a more descriptive name than `Array`? Or is this directly from the spec?